app/src/main/java/me/chester/minitruco/core/EstrategiaGasparotto.java

```diff
@@ -823,23 +830,23 @@ private int joga_primeira_rodada_mao(SituacaoJogo s) {
 
 		if (mandaBala(75)) {
 			// Checa se temos zap e três
-			if (qualidadeCarta(s.cartasJogador[C[0]], s) == ZAP
-					&& qualidadeCarta(s.cartasJogador[C[1]], s) == TRES)
+			if (qualidadeCarta(s.cartasJogador[C[0]], s) == Qualificacao.ZAP.value()
```

Comment:
For example, if we do the change above, this would not need the `.value()`, making it easier to read.